src/core/Akka.Tests/Actor/StashMailboxSpec.cs

```diff
@@ -32,13 +32,12 @@ public void When_creating_actor_marked_with_WithUnboundedStash_a_mailbox_which_s
             intRef.Cell.Mailbox.GetType().ShouldBe(typeof(UnboundedDequeBasedMailbox));
         }
 
-        [Fact(Skip = "We do not have a BoundedDequeBasedMailbox yet")] //TODO: Remove Skip when we have a BoundedDequeBasedMailbox
+        [Fact]
         public void When_creating_actor_marked_with_WithBoundedStash_a_mailbox_which_supports_unbounded_stash_is_created()
         {
             var actorRef = ActorOf<BoundedStashActor>();
             var intRef = (LocalActorRef)actorRef;
-            //intRef.Cell.Mailbox.GetType().ShouldBe(typeof(BoundedDequeBasedMailbox));
-            throw new Exception("Incomplete. Remove the comment on the line above this, and remove this line, when we have BoundedDequeBasedMailbox");
+            intRef.Cell.Mailbox.GetType().ShouldBe(typeof(BoundedDequeBasedMailbox));
```

Comment:
The BoundedDequeBasedMailbox have been ported for quite some time ago.
Test now passes.